curriculum/js2/4.js

```diff
@@ -4,8 +4,11 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, idx = 0, sum = 0) => {
+  if (idx === a.length) {
+    return a.length == 0 ? '' : sum;
```

Comment:
problem states return the sum, which is a number. It's not cool to return a string (not a number)

Reply:
I thought this was a bit weird as well. I'm only returning an empty string because it looks like that's what the test wanted. Okay to modify the test to equal 0 and return zero on the empty array case, or am I missing something?

describe('given an array, return the sum of all the values', () => {
  it('should handle base case of []', () => {
    const result = solution([]);
    expect(result).to.equal('');
  });